components/BottomTabs/styles.module.css

```diff
@@ -0,0 +1,13 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  bottom: 0;
```

Comment:
there is shorthand for top, bottom, right, left... i believe it is called 'inset'